lua/conform/init.lua

```diff
@@ -354,21 +354,24 @@ end
 ---    timeout_ms nil|integer Time in milliseconds to block for formatting. Defaults to 1000. No effect if async = true.
 ---    bufnr nil|integer Format this buffer (default 0)
 ---    async nil|boolean If true the method won't block. Defaults to false. If the buffer is modified before the formatter completes, the formatting will be discarded.
+---    dry_run nil|boolean If true don't edit the buffer, only return whether the buffer would be edited.
```

Comment:
This is a bit deceptive, both because the `did_edit` boolean is available even when this is false and because (per the comment below) we shouldn't actually _return_ `did_edit`. I'd suggest leaving this as
> If true, don't apply formatting changes to the buffer